src/main/java/nextstep/lotto/dto/LottoTicket.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.lotto.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTicket {
+
+    private List<LottoNumber> ticket;
+
+    private LottoTicket(List<LottoNumber> numbers) {
+        this.ticket = new ArrayList<>();
+        for(LottoNumber number : numbers){
+            this.ticket.add(number);
+        }
+    }
+
+    public static LottoTicket create(List<LottoNumber> numbers) {
+        return new LottoTicket(numbers);
+    }
+
+    public int matchCount(List<LottoNumber> winnerLotto) {
+        return (int) ticket.stream()
+                .filter(winnerLotto::contains)
+                .count();
+    }
+
+    public List<LottoNumber> getLottoNumber(){
+        return ticket;
```

Comment:
```suggestion
        return Collections.unmodifiableList(ticket);
```

이렇게 바로 참조를 노출하게 되면 외부에서  변조할 우려가 있습니다~

stream을 이용하거나, 위와 같이 한번 감싸서 외부에서  참조를 수정하는것을 막을 수 있겠네요.

Reply:
Collections.unmodifiableList라는 기능이 있는지 이번에 알게되었습니다.
이 부분을 수정하면서 외부에서 정렬이 불가능 했었는데 이부분도 메소드 안으로 옴기면서 LottoTicket의 자신의 기능이 좀더 강화된거 같습니다.